test/client/getPaths.ts

```diff
@@ -58,41 +56,41 @@ describe("client.getPaths", function () {
   //     this.client.errors.NotFoundError
   //   )
   // })
-  it("invalid PathFind", function () {
-    this.mockRippled.addResponse("ripple_path_find", rippledResponse);
-    assert.throws(() => {
-      this.client.getPaths(REQUEST_FIXTURES.invalid);
-    }, /Cannot specify both source.amount/);
-  });
-  it("does not accept currency", function () {
+  // it("invalid PathFind", function () {
+  //   this.mockRippled.addResponse("ripple_path_find", rippledResponse);
+  //   assert.throws(() => {
```

Comment:
why did this get commented out?

Reply:
It's supposed to fail due to jsonschemas. Maybe it's more correct to delete, but since this method is being heavily rewritten anyways, I didn't think it mattered.